changelog.txt

```diff
@@ -1,5 +1,6 @@
 V.Next
 ----------
+- [MAJOR] Bumped MSAL Broker Protocol version to 9.0, acquireToken/acquireTokenSilent endpoint requires minimum_required_broker_protocol_version of 9.0+ to Send x-ms-PKeyAuth Header to the token endpoint. (#1790)
```

Comment:
https://github.com/AzureAD/ad-accounts-for-android/pull/1927#discussion_r924968243